data/architecture.yml

```diff
@@ -1,135 +1,168 @@
         answers:
             - {value: "%%s?%%s", correct: true}
             - {value: "%s?%s",   correct: false}
             - {value: "?%%s",    correct: false}
             - {value: "?%s",     correct: false}
-    -
-        question: 'Using ClassLoader component, which class can be used to autoload a static class map?'
-        answers:
-            - {value: "Symfony/Component/ClassLoader/MapClassLoader", correct: true}
-            - {value: "Symfony/Component/ClassLoader/ClassMapLoader", correct: false}
-            - {value: "Symfony/Component/ClassLoader/ClassLoader",    correct: false}
-            - {value: "Symfony/Component/ClassLoader/MapLoader",      correct: false}
+        help: |
+             'https://symfony.com/doc/current/reference/configuration/framework.html#version-format'
     -
         question: 'Instantiating a new Symfony\Component\HttpKernel\Kernel, what is the correct arguments order?'
         answers:
-            - {value: "public function __construct($environment, $debug)",               correct: true}
-            - {value: "public function __construct($debug, $environment)",               correct: false}
-            - {value: "public function __construct($name, $environment, $debug)",        correct: false}
-            - {value: "public function __construct($environment, $debug, $name = null)", correct: false}
+            - {value: "public function __construct(string $environment, bool $debug)",               correct: true}
+            - {value: "public function __construct(bool $debug, string $environment)",               correct: false}
+            - {value: "public function __construct(string $name, string $environment, bool $debug)",        correct: false}
+            - {value: "public function __construct(string $environment, bool $debug, string $name = null)", correct: false}
+        help: |
+            'https://github.com/symfony/symfony/blob/4.2/src/Symfony/Component/HttpKernel/Kernel.php'
+
     -
         question: 'What is the prefix of environment variables used by Symfony?'
         answers:
-            - {value: "SYMFONY__", correct: true}
+            - {value: "APP__", correct: true}
+            - {value: "APP--", correct: false}
+            - {value: "SYMFONY__", correct: false}
             - {value: "SF__",      correct: false}
             - {value: "SYMFONY-",  correct: false}
             - {value: "SF-",       correct: false}
+        help: |
+            'no answer found, however, for symfonycloud, the prefix is SYMFONY_ https://symfony.com/doc/current/cloud/cookbooks/env.html#symfonycloud-environment-variables'
     -
-        question: 'In Kernel class, which method do you need to redefine if you want to have multiple configuration files (config_dev.yml, config_test.yml, ...)?'
+        question: 'In Symfony\Component\HttpKernel\Kernel class, which method do you need to redefine if you want to have multiple configuration files (config_dev.yml, config_test.yml, ...)?'
```

Comment:
I think this question should be removed as it is completely obsolete in Symfony. The help text helps, but why the studying person should even bother answering it.